r5dev/sdklauncher/basepanel.cpp

```diff
@@ -641,11 +1275,162 @@ void CUIBaseSurface::VirtualItemToClipboard(const std::unique_ptr<MouseEventArgs
 	clip::set_text(svClipBoard);
 }
 
+//-----------------------------------------------------------------------------
+// Purpose: changes currently showed mod
+// Input  : &pEventArgs - 
+//			*pSender - 
+//-----------------------------------------------------------------------------
+void CUIBaseSurface::ModManagerClick(const std::unique_ptr<MouseEventArgs>& pEventArgs, Forms::Control* pSender) {
+	if (pEventArgs->Button != Forms::MouseButtons::Left)
+		return;
+
+	CUIBaseSurface* pSurface = reinterpret_cast<CUIBaseSurface*>(pSender->FindForm());
+
+	List<uint32_t> lSelected = pSurface->ModsListView->SelectedIndices();
+
+	if (!lSelected.Count())
+		return;
+
+	ModObject object;
+	for (uint32_t i = 0; i < lSelected.Count(); i++)
+		object = pSurface->ModList[lSelected[i]].m_object;
+
+	pSurface->m_ManagerViewerCoverText->Hide();
+	pSurface->m_ManagerViewerNameLabel->SetText(object.name.c_str());
+	pSurface->m_ManagerViewerDescText->SetText(object.description.c_str());
+	std::string authors = "Authors: ";
+	for (int i = 0; i < object.authors.size(); i++) {
```

Comment:
Cast to size_t or use size_t. You did not fix the issues requested by the first review, stopping current review here and requesting fixing everything from first review first before proceeding to second.